### PR TITLE
 [readme first] Cleaned up, fixed item descriptions, added four fallout plants

### PR DESCRIPTION
**Pull request by PlasticCat**
## Description
Might want to comment this one first before the other map changes, all I did is add yucca, punga, coyote tobacco and tato to the rng of wasteland plants but still might want to be safe then let the other changes over right my conflicting file.

## Motivation and Context
makes the code cleaner, removes space references, adds fallout plants, I have gotten many complaints that deathclaws are soft. [Used a line of Trilbyspaceclone code just to avoid a conflict]

## How Has This Been Tested?
commented and hosted, harvested a yucca

## Changelog (necessary)
:cl:
add: yucca
add: tato
add punga
add coyote tobacco
del: unused commented out code
tweak: item descriptions
tweak: made deathclaws tougher and more mean
/:cl:

### CODE REVIEW

**Comment by Covert0ddity** [comment]
Please resolve conflicts.

**Comment by PlasticCat** [comment]
I'm pretty sure the map file conflict is because of me adding the new plants to rng for the deathclaw idk.

**Comment by PlasticCat** [comment]
undid my deathclaw stuff. that can be someone else's problem but the map file..

**Comment by PlasticCat** [comment]
someone please help

**Comment by Covert0ddity** [comment]
you need to redo the stuff you did to the map since we updated to a new format

**Comment by Laus4Deus** [comment]
Overall, removing useless comments is not bad and removing comments such as:
//////// Content list//////
/////////shotgun///////
////////pistol////////
////////arrow///////
Is not terrible either, but on the first 10 file changes or so I see you are removing some comments that explain what the code does and how it works and those are very, very important.

File: code/game/objects/items/airlock_painter.dm [hunk 1]
```diff
@@ -43,53 +43,6 @@
 	else
 		return 1
 
-/obj/item/airlock_painter/suicide_act(mob/user)
```
Comment (Laus4Deus) [inline code/game/objects/items/airlock_painter.dm:46]: Why remove a suicide_act tho
I think toners still exist and can be printed from the lathe

**Pull request closed without merging**